eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/LiquidityAds.scala

```diff
@@ -238,6 +239,9 @@ object LiquidityAds {
 
   case class WillFundPurchase(willFund: WillFund, purchase: Purchase)
 
+  /** Minimal information about a liquidity purchase. */
```

Comment:
Nit, but every time we add a new adjacent class it's some more mental load. We could even rename to `PurchaseInfoForRbf` if it's only used for that (can always rename later).

```suggestion
  /** Minimal information about a liquidity purchase, useful for rbf-ing splice txs. */
```